example/ios/Podfile

```diff
@@ -23,6 +23,23 @@ def flutter_root
   raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
 end
 
+def appodeal
```

Comment:
I didn't understand why you included the Appodeal packages here, in example app's Podfile. Since the example app is using the `appodeal_flutter` plugin then it will automatically install all the packages that are described in the `appodeal_flutter.podspec`.

In other words, if the `appodeal_flutter.podspec` is updated with latest packages from Appodeal then it shouldn't be necessary to update the `example\Podfile`.

Or did I miss something? 🙂

Reply:
No you're right, didn't see they were included there. They are now removed! 👍